proposals/lip-0051.md

```diff
@@ -916,6 +869,8 @@ Let `genesisBlockAssetBytes` be the `data` bytes included in the block assets fo
   * For each `address, tokenID` pair: 
     * The values used as `moduleID` in the `lockedBalances` array must be unique. 
     * The `lockedBalances` array must be in asscending order of `moduleID`.
+    * All elements of the `lockedBalances` array must have `amount != 0`.
+    * `availableBalance != 0` or `lockedBalances != []`, i.e.: the entry must contain some tokens.
```

Comment:
The sentence is incomplete.

Reply:
I added a word, but for me the sentence was quite ok.

Reply:
Now it is clear to me what is meant. But the main clause still does not have a verb.

Reply:
Sure.